graphserver/server.go

```diff
@@ -134,16 +134,35 @@ func (server *ArachneServer) AddGraph(ctx context.Context, elem *aql.ElementID)
 // AddVertex adds a vertex to the graph
 func (server *ArachneServer) AddVertex(ctx context.Context, elem *aql.GraphElement) (*aql.EditResult, error) {
 	var id string
-	server.engine.AddVertex(elem.Graph, *elem.Vertex)
 	id = elem.Vertex.Gid
+	if elem.Update {
+		o := server.engine.GetVertex(elem.Graph, id)
+		if o != nil {
+			for k, v := range o.Data.Fields {
+				if _, ok := elem.Vertex.Data.Fields[k]; !ok {
+					elem.Vertex.Data.Fields[k] = v
```

Comment:
These wouldn't recursively update fields, right?

Reply:
Yes, merging only occurs at the top level.  
Suggested behavior?

Reply:
I guess that's ok. Please add some comments in the python library and proto description to describe that behavior.